martin-mbtiles/src/mbtiles_queries.rs

```diff
@@ -46,7 +47,15 @@ where
        ) AS is_valid;
 "#
     );
-    Ok(sql.fetch_one(&mut *conn).await?.is_valid == 1)
+
+    Ok(sql
+        .fetch_one(&mut *conn)
+        .await?
+        .is_valid
+        .ok_or(MbtError::SqlError(sqlx::Error::ColumnNotFound(
```

Comment:
I'm a bit uncertain why we need explicit error here? Does the new sqlite have some new error response? If not, and this is an option, can we simply assume that its false and treat it as such (i.e. `unwrap_or_default()`)